ocs_ci/framework/main.py

```diff
@@ -108,27 +153,106 @@ def init_ocsci_conf(arguments=None):
         utils.add_path_to_env_path(framework.config.RUN["bin_dir"])
     if args.disable_components:
         framework.config.ENV_DATA["disable_components"] = args.disable_components
-    check_config_requirements()
+
+
+def init_multicluster_ocsci_conf(args, nclusters):
+    """
+    Parse multicluster specific arguments and seperate out each cluster's configuration.
+    Then instantiate Config class for each cluster
+
+    Params:
+        args (list): of arguments passed
+        nclusters (int): Number of clusters (>1)
+
+    """
+    parser = argparse.ArgumentParser(add_help=False)
+    # Dynamically adding the argument --cluster$i to enforce
+    # user's to pass --cluster$i param followed by normal cluster conf
+    # options so that separation of per cluster conf will be easier
+    for i in range(nclusters):
+        parser.add_argument(
+            f"--cluster{i+1}",
+            required=True,
+            action="store_true",
+            help=f"cluster{i}-conf",
+        )
+
+    # Parsing just to enforce `nclusters` number of  --cluster{i} arguments are passed
+    _, _ = parser.parse_known_args(args[2:])
```

Comment:
What error does this raise if the number is not correct? Should we try/except here and raise a more descriptive error / explanation for it?